.github/workflows/test-cli-with-database.yml

```
@@ -59,7 +59,7 @@ jobs:
       - name: Create Medusa project
         working-directory: ..
         run: |
-          medusa new cli-test --skip-db --v2 --branch ci-custom
+          medusa new cli-test --skip-db --skip-migrations --v2 --branch ci-custom
```

self-review: this was silently failing